src/py/flwr/client/rest_client/connection.py

```diff
@@ -80,7 +85,11 @@ def http_request_response(
         """,
     )
 
-    base_url = f"http://{server_address}"
+    base_url = f"https://{server_address}"
+
+    verify: Union[bool, str] = False
```

Comment:
@charlesbvll is `verify=False` the default of the requests library? I would prefer to stick to the default if that makes sense in our case.

Reply:
This is assuming the request uses `https`, right?